recipes/boost/all/patches/1.79.0-geometry_no_rtti.patch

```diff
@@ -0,0 +1,169 @@
+From 379f0c364a5ef3e6d57021ab815ea603ab3f1178 Mon Sep 17 00:00:00 2001
+From: Adam Wulkiewicz <adam.wulkiewicz@gmail.com>
+Date: Tue, 19 Jul 2022 22:52:09 +0200
+Subject: [PATCH] [index] Fix and improve rtree experimental features
```

Comment:
I think this would be a great candidate for #11713 🙏

Reply:
Never mind me https://github.com/conan-io/conan-center-index/pull/11811/files#diff-9caf0a463a8fd05c68a2d642e5dc6592ce8b2b09abfc6ed1d22f4030f6c27252R240-R244